src/client/eppo-client.ts

```diff
@@ -871,7 +871,7 @@ export default class EppoClient {
   getPrecomputedAssignments(
     subjectKey: string,
     subjectAttributes: Attributes | ContextAttributes = {},
-    obfuscated = false,
+    obfuscated = true,
```

Comment:
This is hard coded to be true in other places in the common sdk, e.g.

https://github.com/Eppo-exp/js-sdk-common/blob/7e65d42f8be68c429cf4d08190a467d0543e15ce/src/client/eppo-precomputed-client.ts#L387

So at the very least, the default here should be `true` if we let it support both formats

Reply:
#180 will drop support for this param.